deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
what do you mean by "anything with which I can start the mysql server" ? do you just need command line executables from the mysql package in addition to the library?

there is an unpacked_dir option for Binaries that might help